docs/advanced/ExampleRedditAPI.md

```diff
@@ -174,10 +176,7 @@ export default function configureStore(preloadedState) {
   return createStore(
     rootReducer,
     preloadedState,
-    applyMiddleware(
-      thunkMiddleware,
-      loggerMiddleware
-    )
+    applyMiddleware(thunkMiddleware, loggerMiddleware)
```

Comment:
Here's another case where I'd leave it alone. The line-by-line progression makes more sense before.